app/components/publish-actions.tsx

```diff
@@ -56,13 +56,12 @@ export const TriggeredPopover = (props: TriggeredPopoverProps) => {
     setAnchorEl(trigger);
   }, [trigger]);
 
-  const [ref, width, height] = useResizeObserver<HTMLDivElement>();
+  const [ref, width, height] = useResizeObserver<HTMLElement>();
```

Comment:
Nit: Box returns `HTMLDivElement` by default, should be safe to keep that type 👍